src/main/java/uk/gov/pay/api/model/ValidCreatePaymentRequest.java

```diff
@@ -50,10 +51,12 @@ public String getDescription() {
         return Optional.ofNullable(returnUrl);
     }
 
+    @ApiModelProperty(name = "agreementId", access = "agreementId")
     public Optional<String> getAgreementId() {
         return Optional.ofNullable(agreementId);
     }
 
+    @ApiModelProperty(name = "language", access = "language")
```

Comment:
what has this line got to do with the goal of this story, i.e. "PP-7073 429 response code"?

Reply:
It is part of removing direct debit related changes. We need to define an ApiModelProperty and use it in pom.xml config for swagger-maven-plugin to ignore certain fields.

Reply:
ok 👍